core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/UniqueIDJDBCUserStoreManager.java

```diff
@@ -3223,6 +3224,11 @@ protected UniqueIDPaginatedSearchResult doGetUserListWithID(Condition condition,
         getExpressionConditions(condition, expressionConditions);
 
         for (ExpressionCondition expressionCondition : expressionConditions) {
+            // TODO: This validation has to be removed once ge and le operations are implemented for JDBC userstores.
+            if (ExpressionOperation.GE.toString().equals(expressionCondition.getOperation()) ||
+                    ExpressionOperation.LE.toString().equals(expressionCondition.getOperation())) {
+                throw new UserStoreClientException("ge and le operations are not supported for JDBC userstores.");
```

Comment:
better to define a new error code for this exception

Reply:
ge, le filter implementation for JDBC user stores will be done added with the next PR.

Reply:
https://github.com/wso2/product-is/issues/12845